dotCMS/src/integration-test/java/com/dotcms/publisher/util/DependencyManagerTest.java

```diff
@@ -0,0 +1,256 @@
+package com.dotcms.publisher.util;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+import com.dotcms.contenttype.business.FieldAPI;
+import com.dotcms.contenttype.model.field.Field;
+import com.dotcms.contenttype.model.field.FieldBuilder;
+import com.dotcms.contenttype.model.field.RelationshipField;
+import com.dotcms.contenttype.model.type.ContentType;
+import com.dotcms.datagen.ContentletDataGen;
+import com.dotcms.datagen.TestDataUtils;
+import com.dotcms.publisher.bundle.bean.Bundle;
+import com.dotcms.publisher.bundle.business.BundleAPI;
+import com.dotcms.publisher.business.PublishQueueElement;
+import com.dotcms.publisher.pusher.PushPublisherConfig;
+import com.dotcms.publishing.DotBundleException;
+import com.dotcms.publishing.PublisherConfig.Operation;
+import com.dotcms.util.IntegrationTestInitService;
+import com.dotmarketing.business.APILocator;
+import com.dotmarketing.business.RelationshipAPI;
+import com.dotmarketing.exception.DotDataException;
+import com.dotmarketing.exception.DotSecurityException;
+import com.dotmarketing.portlets.contentlet.business.ContentletAPI;
+import com.dotmarketing.portlets.contentlet.model.Contentlet;
+import com.dotmarketing.portlets.structure.model.ContentletRelationships;
+import com.dotmarketing.portlets.structure.model.ContentletRelationships.ContentletRelationshipRecords;
+import com.dotmarketing.portlets.structure.model.Relationship;
+import com.dotmarketing.util.WebKeys.Relationship.RELATIONSHIP_CARDINALITY;
+import com.google.common.collect.Lists;
+import com.liferay.portal.model.User;
+import java.util.Date;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+/**
+ * @author nollymar
+ */
+public class DependencyManagerTest {
+
+    private static long languageId;
+    private static User user;
+    private static BundleAPI bundleAPI;
+    private static ContentletAPI contentletAPI;
+    private static FieldAPI contentTypeFieldAPI;
+    private static RelationshipAPI relationshipAPI;
+
+    @BeforeClass
+    public static void prepare() throws Exception {
```

Comment:
Codacy found an issue: [A method/constructor should not explicitly throw java.lang.Exception](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5775242)